### PR TITLE
Determine if a FIRApp exists before configuration.

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

File: config/ConfigExampleTests/AppTests.m [hunk 1]
```diff
@@ -27,7 +27,7 @@ @implementation AppTests
 
 
 - (void)testRemoteConfig {
-  [FIRApp configure];
+  if([FIRApp defaultApp] == nil){ [FIRApp configure]; };
```
Comment (paulb777) [inline config/ConfigExampleTests/AppTests.m:30]: To make this scalable, move to the setup function.

Also, fix the style to:

if ([FIRApp defaultApp] == nil) { 
  [FIRApp configure]; 
}

Comment (paulb777) [reply config/ConfigExampleTests/AppTests.m:30]: Here's a Swift example of what I mean https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseRemoteConfig/Tests/SwiftAPI/APITestBase.swift#L28

**Comment by paulb777** [comment]
Please also move to the setUp function so it is shared for any additional tests that get added.

**Comment by granluo** [comment]
Yep, FIRApp seems to be configured in appdelegate.
https://github.com/firebase/quickstart-ios/search?p=1&q=firapp+configure&unscoped_q=firapp+configure

will test the remoteconfig and populate to the rest quickstarts.

**Pull request merged**